spring-integration-jpa/src/main/java/org/springframework/integration/jpa/outbox/DefaultOutboxEventSender.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.jpa.outbox;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * {@link OutboxEventSender} interface implementation that converts the {@link ExportEvent} into {@link OutboxEvent}
+ * entity and persists it in the outbox table via the {@link OutboxEventRepository} repository.
+ *
+ * @author Christian Tzolov
+ *
+ * @since 6.2
+ */
+public class DefaultOutboxEventSender implements OutboxEventSender {
```

Comment:
I don't remember that we have agreed to apply these changes.
Perhaps you have messed up some branches, so unexpected files has slipped to your current proposal?..